packages/zapp/console/src/components/Launch/LaunchForm/ResumeFormDialog.tsx

```diff
@@ -0,0 +1,41 @@
+import { Dialog } from '@material-ui/core';
```

Comment:
This one is ~90% similarity, so no need for a separate component too.
Next props would be optional: `id`, `compiledNode`, `nodeId`, so inside the `Dialog` you'll have a ternary `id ? <LaunchForm /> : {compiledNode && nodeId && <ResumeForm />}` or something like that